app/grandchallenge/algorithms/tasks.py

```diff
@@ -21,6 +23,100 @@
 from grandchallenge.subdomains.utils import reverse
 
 
+@shared_task
+def create_algorithm_job_for_inputs(
+    *, algorithm_image_pk, civ_pks, upload_pks, creator_pk
+):
+    try:
+        algorithm_image = AlgorithmImage.objects.get(pk=algorithm_image_pk)
+    except AlgorithmImage.DoesNotExist:
+        return
```

Comment:
Should this pass silently?

Reply:
I assumed so, given the existing `test_no_algorithm_image_does_nothing`. We could also raise an exception of course.

Reply:
I think it should error out here so fine to remove the try except, it would be really strange if an algorithm image was used and then it is gone at this point, we should be aware of that. I will take a look at the existing test once this PR is done.